Integrations/integration-SplunkPy.yml

```diff
@@ -67,6 +72,20 @@ script:
     import urllib3
     urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 
+    def rawToDict(raw):
+        result = {}
+        raw = raw.strip("}")
```

Comment:
Since format may defer add some checks / error handling, or wrap in a try catch

Reply:
strip doesn't fail (throw error)